config/printer-sovol-sv06-2022.cfg

```diff
@@ -0,0 +1,172 @@
+
+# Flash this firmware by copying "out/klipper.bin" to a SD card and
+# turning on the printer with the card inserted. The firmware
+# filename must end in ".bin" and must not match the last filename
+# that was flashed.
+
+# See docs/Config_Reference.md for a description of parameters.
+[include mainsail.cfg]
+
+[mcu]
+serial: /dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0
+restart_method: command
+
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 3000
+max_z_velocity: 5
+max_z_accel: 100
+
+[stepper_x]
+step_pin: PC2
+dir_pin: !PB9
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: tmc2209_stepper_x:virtual_endstop
+position_endstop: 0
+position_max: 225
+homing_speed: 40
+homing_retract_dist: 0
+
+[tmc2209 stepper_x]
+uart_pin: PC1
+run_current: 0.860
+stealthchop_threshold: 180
+interpolate: False
+sense_resistor: 0.150
+uart_address: 3
+driver_SGTHRS: 81
+diag_pin: PA5
+
+[stepper_y]
+step_pin: PB8
+dir_pin: PB7
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: tmc2209_stepper_y:virtual_endstop
+position_endstop: 0
+position_max: 225
+homing_speed: 40
+homing_retract_dist: 0
+
+[tmc2209 stepper_y]
+uart_pin: PC0
+run_current: 0.900
+stealthchop_threshold: 180
+interpolate: False
+sense_resistor: 0.150
+uart_address: 3
+driver_SGTHRS: 82
+diag_pin: PA6
+
+[stepper_z]
+step_pin: PB6
+dir_pin: !PB5
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 4
+endstop_pin: probe:z_virtual_endstop
+position_min: -4
+position_max: 261
+homing_speed: 4
+
+[tmc2209 stepper_z]
+uart_pin: PA15
+run_current: 1.000
+stealthchop_threshold: 40
+interpolate: False
+sense_resistor: 0.150
+uart_address: 3
+diag_pin: PA7
+
+[thermistor sovol_thermistor]
+temperature1: 25
+resistance1: 94162
+beta: 4160
+
+[extruder]
+max_extrude_only_distance: 100.0
+step_pin: PB4
+dir_pin: !PB3
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 4.56
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PA1
+sensor_type: sovol_thermistor
+sensor_pin: PC5
+control: pid
+pid_kd: 86.991
+pid_kp: 21.479
+pid_ki: 1.326
+min_temp: 0
+max_temp: 300
+pressure_advance: 0.023
+
+[tmc2209 extruder]
+uart_pin: PC14
+run_current: 0.550
+stealthchop_threshold: 60
+interpolate: False
+sense_resistor: 0.150
+uart_address: 3
+
+[heater_bed]
+heater_pin: PA2
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PC4
+control: pid
+pid_kp: 64.440
+pid_ki: 0.773
+pid_kd: 1343.571
+min_temp: 0
+max_temp: 130
+
+[fan]
+pin: PA0
+
+[probe]
+pin: PB1
+x_offset: 28
+y_offset: -20
+z_offset: 0
+samples: 2
+samples_tolerance: 0.015
+samples_tolerance_retries: 5
+
+[safe_z_home]
+home_xy_position: 85,135
+z_hop: 10
+z_hop_speed: 5
+
+[bed_mesh]
+speed: 120
+horizontal_move_z: 5
+mesh_min: 28, 20
+mesh_max: 197, 205
+probe_count: 10
+algorithm: bicubic
+fade_start: 1
+fade_end: 10
+fade_target: 0
+
+[display]
+lcd_type: st7920
+cs_pin: PB12
+sclk_pin: PB13
+sid_pin: PB15
+encoder_pins: ^PB14, ^PB10
+click_pin: ^!PB2
+
+[input_shaper]
```

Comment:
Input shaper should also not be configured for a base template for this printer, remove this section

Reply:
removed